cmake/GoogleCloudCppFeatures.cmake

```diff
@@ -241,34 +241,35 @@ set(GOOGLE_CLOUD_CPP_COMPUTE_LIBRARIES
     "compute_region_operations"
     "compute_zone_operations")
 
-list(FIND GOOGLE_CLOUD_CPP_EXPERIMENTAL_LIBRARIES "compute"
-     compute_experimental)
-if (NOT compute_experimental EQUAL -1)
-    list(REMOVE_ITEM GOOGLE_CLOUD_CPP_EXPERIMENTAL_LIBRARIES "compute")
-    list(APPEND GOOGLE_CLOUD_CPP_EXPERIMENTAL_LIBRARIES
-         ${GOOGLE_CLOUD_CPP_COMPUTE_LIBRARIES})
-    list(SORT GOOGLE_CLOUD_CPP_EXPERIMENTAL_LIBRARIES)
-endif ()
+# Use a function to get a new scope, so the GOOGLE_CLOUD_CPP_*_LIBRARIES remain
+# unchanged.
+function (export_libraries_bzl)
+    if ("compute" IN_LIST GOOGLE_CLOUD_CPP_EXPERIMENTAL_LIBRARIES)
```

Comment:
Q.  Is it possible to indirect the variable names so that we could loop over them?

Reply:
Good idea, done.